tests/fe/fe_enriched_step-36.cc

```diff
@@ -740,14 +640,14 @@ int main (int argc,char **argv)
 
   try
     {
-      dealii::Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
+      Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
       {
         Step36::EigenvalueProblem<dim> step36;
-        dealii::PETScWrappers::set_option_value("-eps_target","-1.0");
-        dealii::PETScWrappers::set_option_value("-st_type","sinvert");
-        dealii::PETScWrappers::set_option_value("-st_ksp_type","cg");
-        dealii::PETScWrappers::set_option_value("-st_pc_type", "gamg");
-        dealii::PETScWrappers::set_option_value("-st_ksp_tol", "1e-11");
+        PETScWrappers::set_option_value("-eps_target","-1.0");
+        PETScWrappers::set_option_value("-st_type","sinvert");
+        PETScWrappers::set_option_value("-st_ksp_type","cg");
+        PETScWrappers::set_option_value("-st_pc_type", "gamg");
+        PETScWrappers::set_option_value("-st_ksp_tol", "1e-11");
```

Comment:
I am getting the warning:
```
+ WARNING! There are options you set that were not used!
+ WARNING! could be spelling mistake, etc!
+ Option left: name:-st_ksp_tol value: 1e-11
```
when running the test (without your PR). Do you know why?